dart/dynamics/UniversalJoint.h

```diff
@@ -54,6 +54,9 @@ class UniversalJoint : public MultiDofJoint<2>
   UniversalJoint(const Eigen::Vector3d& _axis0 = Eigen::Vector3d::UnitX(),
                  const Eigen::Vector3d& _axis1 = Eigen::Vector3d::UnitY(),
                  const std::string& _name = "Universal joint");
+  // TODO(MXG): We should be more consistent about indexing starting from 0 or 1
+  // In the constructor arguments, we start with axis0, but in the member
+  // functions we start with axis1.
```

Comment:
Wait, I thought you mentioned about the name affix for DegreeOfFreedom of UniversalJoint, but it was about setAxis1() and setAxis2() (and corresponding getters).

I looked up to find other names have numbers. For example, [Contact](https://github.com/dartsim/dart/blob/master/dart/collision/CollisionDetector.h#L60) and [ContactConstraint](https://github.com/dartsim/dart/blob/master/dart/constraint/ContactConstraint.h#L56) has members indexing starting from 1. So let's change the parameter names to _axis1 and _axis2. It will not change the API as well.